app/webpacker/components/RegistrationsV2/RegistrationAdministration/RegistrationActions.jsx

```diff
@@ -16,9 +16,7 @@ import {
 import { useConfirm } from '../../../lib/providers/ConfirmProvider';
 
 function V3csvExport(selected, registrations, competition) {
-  let csvContent = 'data:text/csv;charset=utf-8,';
```

Comment:
Sidenote, why did this MIME type and UTF-8 preamble have to go?
I feel like the answer is "because we are generating it outside of this method now". But then, how is it not completely broken because the header would appear twice (once generated outside and once inside the method) right now??

Reply:
so it is completely broken right now...